src/view/hooks/useUploadPlaceImage.ts

```diff
@@ -97,6 +119,12 @@ const useUploadPlaceImage = () => {
 
                     const storageRef = ref(storage, `images/${uniqueFileName}`);
                     const { downloadUrl } = await uploadFile(file, storageRef);
+
+                    const hash = generateFileHash(file);
+                    setUploadedImageHashes(
+                        new Set(uploadedImageHashes.add(hash))
+                    );
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add#return_value
返り値がSetなので以下のようにしても良さそうです？
```suggestion
                    setUploadedImageHashes(uploadedImageHashes.add(hash));
```